src/frontend/model/frontend_store.rs

```diff
@@ -7,4 +7,5 @@ pub struct FrontendStore {
     pub cached_sounds: HashMap<String, String>,
     pub audio_ref: NodeRef<leptos::html::Audio>,
     pub error: RwSignal<Vec<String>>,
+    pub error_soft: RwSignal<Vec<String>>,
```

Comment:
The FrontendStore essentially already is wrapped in a `RwSignal` so I don't think the extra `RwSignal` is needed